snuba/datasets/configuration/discover/entities/discover.yaml

```diff
@@ -28,6 +28,7 @@ schema:
     },
     { name: geo_region, type: String, args: { schema_modifiers: [nullable] } },
     { name: geo_city, type: String, args: { schema_modifiers: [nullable] } },
+    { name: geo_subdivision, type: String, args: { schema_modifiers: [nullable] } },
```

Comment:
why is this column necessary if you have a mapper. Did something fail because you didn't have this?

Reply:
i'm mostly replicating what we do for the rest of geo information tbh

Reply:
I'm just trying to fix the errors I have here https://github.com/getsentry/sentry/pull/47995

Reply:
https://github.com/getsentry/sentry/actions/runs/4813125117/jobs/8569256186?pr=47995

Reply:
also when I tried querying for geo.subdivision it didn't return any data although there were events that had it set when I look at their json

Reply:
so i'm wondering if i missed something in my last pr

Reply:
I see, I don't think this column addition is necessary and the way to check for sure would be to write an api test like [here](https://github.com/getsentry/snuba/blob/master/tests/test_snql_api.py#L55) which should give you the confidence that all the columns are where they should be.

Also side question, are these columns only going to be queried through discover?